hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/Federation.md

```diff
@@ -536,7 +536,7 @@ To enable cross-origin support (CORS) for the Yarn Router, please set the follow
 #### How to configure Router Cache
 
 Cache is enabled by default. When we set the `yarn.federation.cache-ttl.secs` parameter and its value is greater than 0, Cache will be enabled.
-We currently provide two Cache implementations: `JCache` and `GuavaCache`.
+We currently provide Three Cache implementations: `JCache` , `GuavaCache`, `CaffeineCache`
```

Comment:
`three` with lower case.

Reply:
Thanks for your suggestions! I will improve the documentation.